www.foia.gov/_sass/modules/_glossary.scss

```diff
@@ -0,0 +1,104 @@
+// Borrowed from FEC
+$background-color: $color-primary-darkest;
+$z-glossary: 9000; // uswds navigation
+
+.glossary {
+  @include transition(right, .3s);
+  background-color: $background-color;
+  bottom: 0;
+  color: $color-gray-lightest;
+  max-width: 40rem;
+  overflow-y: scroll;
+  padding: 3rem;
+  position: fixed;
+  top: 0;
+  width: 75%;
+  z-index: $z-glossary;
```

Comment:
oh, clever, I like that

Reply:
:+1: I'm not really familiar with the uswds scss files, so I'm not sure if they have their own set of variables for this kind of thing. I was surprised to see the z-index so high on the uswds nav.

Reply:
They don't really have `z-index` variables, which I think is kind of surprising. I like being able to track it in one spot!